File: testdata/projects/php/tests/MathTest.php
<?php

namespace Example\Tests;

use Example\Math;
use PHPUnit\Framework\TestCase;

class MathTest extends TestCase
{
    public function testAdd(): void
    {
        $math = new Math();
        $this->assertSame(5, $math->add(2, 3));
    }

    public function testIsPositive(): void
    {
        $math = new Math();
        $this->assertTrue($math->isPositive(1));
        $this->assertFalse($math->isPositive(-1));
    }
}

File: testdata/projects/java/src/test/java/example/MathTest.java
package example;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class MathTest {
    @Test
    void testAdd() {
        assertEquals(5, Math.add(2, 3));
    }

    @Test
    void testIsPositive() {
        assertTrue(Math.isPositive(1));
        assertFalse(Math.isPositive(-1));
    }
}
